src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
Header comment is necessary for overriden methods to tell other what the overriding is for. Add the comments. Same for below.